src/platforms/esp32/test/main/test_erl_sources/test_crypto.erl

```diff
@@ -207,6 +210,43 @@ test_available_ciphers() ->
         crypto:crypto_one_time(aes_128_ecb, <<1:128>>, <<2:128>>, <<3:128>>, false),
     <<172, 173, 71, 170, 66, 92, 132, 117, 22, 33, 191, 18, 17, 207, 171, 236>> =
         crypto:crypto_one_time(aes_192_ecb, <<1:192>>, <<2:128>>, <<3:128>>, false),
+    <<33, 51, 81, 23, 26, 72, 178, 26, 115, 82, 208, 26, 225, 24, 76, 247>> =
+        crypto:crypto_one_time(aes_256_ecb, <<1:256>>, <<2:128>>, <<3:128>>, false),
+    ok;
+test_available_ciphers(_Model) ->
```

Comment:
I think we should discuss about enabling aes-192 with software implementation, but this is a side discussion.

Reply:
https://github.com/atomvm/AtomVM/issues/1108